lib/utils/app_list.dart

```diff
@@ -14,18 +14,21 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+import 'apps_stub.dart' if (dart.library.io) 'package:files/main.dart';
+// ignore: duplicate_import
+import 'apps_stub.dart' if (dart.library.io) 'package:terminal/main.dart';
```

Comment:
Basically what I do here is to import 'apps_stub.dart', but if we're running on dart:io, import both Files and Terminal.